src/flash/com/longtailvideo/jwplayer/player/Player.as

```diff
@@ -63,16 +64,36 @@ public class Player extends Sprite implements IPlayer {
         // e.state can be ThrottleType.THROTTLE, ThrottleType.PAUSE, or ThrottleType.RESUME
         // in Chrome we only get 'throttle' and 'resume' for offscreen and power-save throttling
         var state:String = e['state'] as String;
-        if (state === 'resume') {
-            // After resume, we're past Chrome Power Save
-            // stop listening to the off-screen throttle events
-            this.removeEventListener('throttle', flashThrottled);
+
+        if (state !== 'resume') {
+            _model.removeEventListener(MediaEvent.JWPLAYER_MEDIA_TIME, removeThrottleListener);
+            // Ignore throttle events for players ignored by Chrome's heuristics

```

Comment:
We should be consistent about what level of abstraction to use inside a specific statement. I'd prefer to see this chrome stuff higher up, perhaps in providers/flash.js where we write 

```js
                    // We cannot rely on firefox to properly return results
                    if (utils.isChrome()) {
                        _swf.on('throttle', onThrottle, this);
                    }
```